agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -454,6 +454,8 @@ bool slave_thread::handle_cmdu_control_ieee1905_1_message(Socket *sd,
     master_last_seen   = std::chrono::steady_clock::now();
     keep_alive_retries = 0;
 
+    auto rx_mid = cmdu_rx.getMessageId();
```

Comment:
This doesn't deserve its own commit, do this when you use it.

Reply:
fixed